packages/ens-controller/CHANGELOG.md

```diff
@@ -7,6 +7,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Changed
```

Comment:
Are we releasing this package? If not, should we add these changelog entries in another PR?

Reply:
To include the changes in https://github.com/MetaMask/core/pull/4556, I added releases for:

- `ens-controller` (https://github.com/MetaMask/core/pull/4548/commits/a3308b8714a82c6e763188d085869be0f01c5a4d)
- `gas-fee-controller` (https://github.com/MetaMask/core/pull/4548/commits/ac917f03b368be2addc5054632c052ce96d5f662)
- `selected-network-controller`, `queued-request-controller` https://github.com/MetaMask/core/pull/4548/commits/61670f388a787cbbb6e96215fd57a3b3284ad98c
- `transaction-controller`, `user-operation-controller` https://github.com/MetaMask/core/pull/4548/commits/773baa3423515591579987f7fa79c793a5adebd4

Every package with a diff in this PR is now being released (excluding `json-rpc-engine`).